docs/man/output.1

```diff
@@ -90,7 +90,21 @@ timestamp for events.
 file hash (sha256).
 .RS 2
 .IP \[bu] 2
-Affected events: sched_process_exec, shared_object_loaded
+Affected events: \f[I]sched_process_exec\f[R],
+\f[I]shared_object_loaded\f[R]
+.IP \[bu] 2
+\f[B]inode\f[R] option recalculates the file hash if the inode\[cq]s
+creation time (ctime) differs, which can occur in different namespaces
+even for identical pathnames.
```

Comment:
Maybe add a note that this should reduce performance in the case of deploying the same image for multiple containers?
It should be clear that this is the preferred method for these cases if the container enrichment is not available